Development/pcm.yml

```diff
@@ -0,0 +1,43 @@
+name: ConsIndShock
+
+symbols: # section where all symbols used in the model must be defined
+  states: [m]
+  controls: [c, stigma]
+  distributions: [perm, tran, risky]
+  parameters: [beta, rho, R, G, std_perm, std_tran, mean_risky, std_risky]
+
+domain: # section specifying the boundaries for the state variables
+  m: [0, .inf]
+  c: [0, .inf]
+  stigma: [0, 1]
+
+equations: # section containing the list of equations
+  transitions:
+    m_: R_port * a / (G * perm) + tran

```

Comment:
state transition

Reply:
what is the meaning of the underscore in `m_`?

Reply:
Need to talk to @mnwhite about how he's doing this, since we don't want any references to `next` in our language. `w` is a reserved function for `continuation`, for now I'm using `m_` as the stand-in for `m_{t+1}`.